### ISSUE TITLE
React-router 1.0.2 has a peer dependency for history 1.13.x, but current is 1.16.x

### ISSUE THREAD

**Issue by Pomax** [rackt/react-router#2742]
Could this be changed to `^1.13.x` so that npm install doesn't report failures during the install process due to unmet peer dependencies?

**Comment by taion** [comment]
1.16.0 has a wrong deprecation warning, and otherwise 1.0.2 will spew unavoidable deprecation warnings against 1.14.x and 1.15.x.

We need to cut `history` 1.16.1 and a corresponding Router 1.0.3 before we can do anything reasonable here. There's been a lot of issues filed on this recently - please take a quick search before you open new ones in the future.

**Comment by Pomax** [comment]
I did. [searching for history in the issues](https://github.com/rackt/react-router/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+history) didn't find anything specifically about a problem with history in the current version, only about future planning for history 2.0, so I filed a new issue. And with this one closed, the next person will probably not find anything again, either =)
